### PR TITLE
Merchant invoices

### PR DESCRIPTION
**Pull request by boomclear**
Has all of merchant-items included, completes merchant-invoices stories. 

100% coverage.

### CODE REVIEW

File: app/views/merchants/invoices/show.html.erb
Comment (garrettgregor) [inline app/views/merchants/invoices/show.html.erb:1]: nice formatting!!!

File: config/routes.rb
Comment (garrettgregor) [inline config/routes.rb:1]: nice use of resources!

**Review by garrettgregor — APPROVED** [review]
Looks great! ready to deploy!

**Review by JCBeldo — APPROVED** [review]
Looks good!

File: app/models/invoice.rb [hunk 1]
```diff
@@ -14,4 +14,12 @@ def self.sorted_incomplete_invoices
     .order(:created_at)
   end
 
+  def customer_full_name
+    "#{customer.first_name} #{customer.last_name}"
+  end
+
+  def total_revenue
+    invoice_items.sum("(quantity * unit_price )/ 100.0").round(2).to_s
```
Comment (jcjurado3) [inline app/models/invoice.rb:22]: nice solution here.

File: app/views/merchants/invoices/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<h2>Invoice #<%= @invoice.id %></h2>
+<p>Status: <%= @invoice.status %></p>
+<p>Created on: <%= @invoice.format_time_stamp %></p>
+<p>Total Revenue: $<%= @invoice.total_revenue %></p>
+<h2>Customer:</h2>
+<p><%= @invoice.customer_full_name%></p>
+<style>
+table, th, td {
```
Comment (jcjurado3) [inline app/views/merchants/invoices/show.html.erb:8]: great use of the table.

**Review by jcjurado3 — APPROVED** [review]

**Pull request merged**